quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/command/ShowConfig.java

```diff
@@ -53,7 +53,7 @@ public final class ShowConfig extends AbstractCommand implements Runnable {
 
     public static final String NAME = "show-config";
     private static final List<String> ignoredPropertyKeys = List.of(
-            "kc.config.args", "kc.show.config", "kc.profile", "kc.quarkus-properties-enabled", "kc.home.dir");
```

Comment:
Just out of curiosity. We're still setting the `kc.config.args` sys prop as before. Why we don't need to ignore it any longer? It might still be recognized as a config option, no?

Reply:
We must have at some point seen system properties as config properties. This and the check for build time properties seem to indicate that - however system properties are no longer picked up as config properties, so we don't need this.